packages/wp-build/src/wordpress-externals-plugin.mjs

```diff
@@ -220,6 +220,7 @@ export function createWordpressExternalsPlugin(
 								return {
 									path: args.path,
 									external: true,
+									sideEffects: !! packageJson.sideEffects,
```

Comment:
This tells the bundler whether the external import has side effects, and allows the bundler to eliminate the import completely.

This simple code is rather blunt. If a package declares only `*.scss` styles as side-effectful, which is rather common, we'll treat the entire package, including its JS code, as side-effectful.

It works for now, but should be expanded into really checking the list of file patterns.